synapse/config/repository.py

```diff
@@ -202,6 +215,21 @@ def generate_config_section(self, data_dir_path, **kwargs):
 
         return (
             r"""
```

Comment:
can you remove the spurious newline while you are here:

```suggestion
            r"""\
```